Knoema.Client/Search/TimeseriesSearch/TimeseriesSearchFacets.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Linq;
+
+namespace Knoema.Search.TimeseriesSearch
+{
+	public class TimeseriesSearchFacets
+	{
+		public TimeseriesSearchDataSourceInfos DataSourceInfos { get; set; }
+		public Dictionary<string, int> DataSources { get; set; }
+		public int Forecasts { get; set; }
+		public Dictionary<string, int> Frequencies { get; set; }
+		public Dictionary<string, int> LastUpdates { get; set; }
+		public int LongTimeseries { get; set; }
+		public TimeseriesSearchRegionLinkInfos RegionLinkInfos { get; set; }
```

Comment:
special class is redundant, it can be just a Dictionary<string, GeographyRegionViewModel>

Reply:
done